### Question 1

In statistics, measures of central tendency represent the middle or center of a set of data values. The three most common measures of central tendency are:

1. Mean: The sum of all values divided by the total number of values.
2. Median: The middle number in an ordered data set.
3. Mode: The most frequent value.

### Question 2

The difference between mean, median and mode are: Mean is the average value of the given observations. Median is the middle value of the given observations. Mode is the most repeated value in the given observation.

how are they calculated :-

1. Mean -  It's calculated by adding all the numbers in the set and dividing by the number of values.
2. Median - The middle value in a data set when it's ordered from least to greatest.
3. Mode - The number that appears most often in a data set. 

### Question 3

Height data: [178, 177, 176, 177, 178.2, 178, 175, 179, 180, 175, 178.9, 176.2, 177, 172.5, 178, 176.5]

1. Mean of the data:-

    total sum/total numbers
2646.3/16 = 177.01
mean = 177.01

2. Median of the data :-
    To find the median, we first need to arrange the data in ascending order.
since the number of values is even,
median = (177+177)/2 = 177
median = 177

3. Mode of the data :-
    There are 2 modes for the given dataset, it is also known as bimodal.
mode = 177,178

### Question 4

In [1]:
import numpy as np


In [2]:
data  = [178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5] 


In [5]:
np.std(data)

1.7885814036548633

Standard Deviation = 1.7885814036548633

### Question 5

1. Range
    The range is the difference between the largest and smallest data points in a set. For example, if a dataset has values of 1, 2, 3, 4, and 5, the range would be 5 - 1 = 4. The range is a simple measure of dispersion, but it can be misleading if the dataset has a few outliers.
    
2. Variance
    The variance is a more sophisticated measure of dispersion that takes into account all of the data points in a set. To calculate the variance, you first need to find the mean of the dataset. Then, you subtract the mean from each data point and square the difference. Finally, you add up all of the squared differences and divide by the number of data points. The variance is a measure of how much the data points deviate from the mean.

3. Standard Deviation
    The standard deviation is the square root of the variance. The standard deviation is a measure of how spread out the data points are around the mean. A small standard deviation indicates that the data points are clustered close to the mean, while a large standard deviation indicates that the data points are more spread out

### Question 6


A Venn diagram is a diagram that uses overlapping circles or other shapes to show the logical relationships between two or more sets of items.

### Question 7

(i)  i cannot understand A B. So, i did not tried this. Please mention in the results what i have to do.

(ii)
A = (2,3,4,5,6,7) 
B = (0,2,6,8,10)
 A U B = (2,3,4,5,6,7,0,8,10)

### Question 8

Skewness is a statistical measure of the asymmetry or distortion of a data set. It's a measure of the deviation of a random variable's given distribution from a symmetric distribution, such as a normal distribution.

### Question 9 

If the data is right skewed i.e +ve skewed then:
mean>median

### Question 10

Both covariance and correlation measure the relationship and the dependency between two variables. Covariance indicates the direction of the linear relationship between variables. Correlation measures both the strength and direction of the linear relationship between two variables. Correlation values are standardized.

### Question 11

The general formula for calculating the sample mean is given by x̄ = ( Σ xi ) / n. Here, x̄ represents the sample mean, xi refers all X sample values and n stands for the number of sample terms in the data set. 

Example dataset:-
data = [1,2,4,7,6]
x̄ = (1+2+4+7+6)/5
x̄ = 4

### Question 12

In a normal distribution, data is symmetrically distributed with no skew. Most values cluster around a central region, with values tapering off as they go further away from the center. The mean, mode and median are exactly the same in a normal distribution.

[i.e. mean = median = mode]

### Question 13

 Covariance indicates the direction of the linear relationship between variables. Correlation measures both the strength and direction of the linear relationship between two variables

### Question 14

Outliers can significantly impact measures of central tendency and dispersion.

Outliers are numbers in a data set that are vastly larger or smaller than the other values in the set. Mean is the only measure of central tendency that is always affected by an outlier.

e.g - dataset = [0.1,0.12,0.45,0.78,0.015,1000]
Now here 1000 is outlier for the dataset.